charts/dscp-matchmaker-api/ci/ct-values.yaml

```diff
@@ -0,0 +1,25 @@
+node:
+  enabled: true
+  node:
+    chain: dev
+    flags:
+    - "--rpc-external"
+    - "--alice"
+    - "--ws-external"
+    - "--rpc-methods=Unsafe"
+    - "--rpc-cors=all"
+    - "--unsafe-ws-external"
+    - "--unsafe-rpc-external"
+userUri: 5FHneW46xGXgs5mUiveU4sbTyGBzmstUspZC92UhjJM694ty
+livenessProbe:
+  enabled: false
+readinessProbe:
+  enabled: false
+startupProbe:
+  enabled: false
```

Comment:
If these are disabled top level are we actually testing anything?

Reply:
I see no other way to test this without adding ipfs and the identity service as deps. An alternative would be to create a service which mocks the health checks polls to ipfs/identity service.

Reply:
what if we create a couple of envs that configure the healthcheck? That way we could disable those aspects of the health check for this purpose. Backlog ticket?

Reply:
Yeah that'll work.